provisioner/terraform/tfparse/funcs.go

```diff
@@ -0,0 +1,162 @@
+package tfparse
+
+import (
+	"github.com/aquasecurity/trivy-iac/pkg/scanners/terraform/parser/funcs"
+	"github.com/hashicorp/hcl/v2/ext/tryfunc"
+	ctyyaml "github.com/zclconf/go-cty-yaml"
+	"github.com/zclconf/go-cty/cty"
+	"github.com/zclconf/go-cty/cty/function"
+	"github.com/zclconf/go-cty/cty/function/stdlib"
+	"golang.org/x/xerrors"
+)
+
+// Functions returns a set of functions that are safe to use in the context of
+// evaluating Terraform expressions without any ability to reference local files.
+// Functions that refer to file operations are replaced with stubs that return a
+// descriptive error to the user.
+func Functions() map[string]function.Function {
+	return allFunctions
```

Comment:
Using `allFunctions` here is confusing; the description above says "set of functions that are safe to use", but `allFunctions` contains unsafe functions.

Reply:
Renamed to avoid confusion.